teku/src/test/java/tech/pegasys/teku/cli/options/P2POptionsTest.java

```diff
@@ -472,4 +480,58 @@ public void defaultPortsAreSetCorrectly() {
     assertThat(networkConfig.getAdvertisedPort()).isEqualTo(DEFAULT_P2P_PORT);
     assertThat(networkConfig.getAdvertisedPortIpv6()).isEqualTo(DEFAULT_P2P_PORT_IPV6);
   }
+
+  @Test
+  public void staticPeersUrl_shouldReadPeersFromUrl(@TempDir final Path tempDir) throws Exception {
+    // Create a test file with peers
+    final Path peersFile = tempDir.resolve("static-peers.txt");
+    Files.writeString(peersFile, "peer1\npeer2\n#comment\n\npeer3");
```

Comment:
NIT: For the sake of completeness, we could add an empty line here as well.

Reply:
it has a blank line before peer3